backend/src/canvas_state.erl

```diff
@@ -0,0 +1,25 @@
+-module(canvas_state).
+
+-export([start/0]).
+
+
+server_loop(State) ->
+    receive
+        % blocks should be a JSON compatible map
```

Comment:
Wait isn't it an array of objects? (array of maps I guess) or am I interpreting this wrong